MailCore/Cache/AccountManager.swift

```diff
@@ -489,4 +489,10 @@ public final class AccountManager: RefreshTokenDelegate, ObservableObject {
             bugTracker.stopActivatingOnScreenshot()
         }
     }
+    
+    public func updateConversationSettings() {
+        for mailboxManager in mailboxManagers.values {
```

Comment:
This will only work for already instantiated mailboxmanagers. You have to loop and create each mailboxManager an example can be seen here -> NotificationsHelper.swift:71